client/ayon_deadline/plugins/publish/global/collect_deadline_job_env_vars.py

```diff
@@ -36,16 +36,16 @@ class CollectDeadlineJobEnvVars(pyblish.api.ContextPlugin):
     ]
 
     def process(self, context):
-        env = {}
+        env = context.data.setdefault(JOB_ENV_DATA_KEY, {})
         for key in self.ENV_KEYS:
+            # Skip already set keys
+            if key in env:
```

Comment:
Should we debug log this maybe if the value differs?

Reply:
At the end we should get rid of the plugin, so I don't think it is necessary.